Index: chromium-131.0.6778.85/build/rust/cargo_crate.gni
===================================================================
--- chromium-131.0.6778.85.orig/build/rust/cargo_crate.gni
+++ chromium-131.0.6778.85/build/rust/cargo_crate.gni
@@ -432,7 +432,7 @@ template("cargo_crate") {
         ]
       }
       if (current_cpu == "arm64" || current_cpu == "x64" ||
-          current_cpu == "loong64" || current_cpu == "riscv64") {
+          current_cpu == "loong64" || current_cpu == "riscv64" || current_cpu == "ppc64") {
         args += [
           "--pointer-width",
           "64",
